codegen/codegen-core/src/main/java/software/amazon/smithy/java/codegen/CodegenUtils.java

```diff
@@ -255,6 +255,19 @@ public static String toDefaultValueName(String memberName) {
         return CaseUtils.toSnakeCase(memberName).toUpperCase(Locale.ENGLISH) + "_DEFAULT";
     }
 
+    /**
+     * Gets the name to use when defining the default value of a member.
+     *
+     * @param memberShape memberShape.
+     * @return Upper snake case name of default
+     */
+    public static String toGetterName(MemberShape memberShape, String memberName) {
+        var prefix = memberShape.isBooleanShape() ? "is" : "get";
```

Comment:
`memberShape.isBooleanShape()` is always false because it's a MemberShape. You need to see the shape type of its target.